src/specimens/Type.js

```diff
@@ -84,25 +84,30 @@ class Type extends React.Component {
 
     const headings = options.headings
       ? options.headings.map( (heading, i) => {
-        const isPixel = typeof heading === 'number' ? 'px' : '';
+        const headingValue = typeof heading === 'object' ? heading.value : heading;
```

Comment:
Here (and in all subsequent cases) please also check for `heading !== null` because `typeof null === 'object'` (you could also filter headings before mapping).